internals/cli/cmd_help.go

```diff
@@ -150,7 +146,7 @@ func (cmd cmdHelp) Execute(args []string) error {
 	}
 	if cmd.All {
 		if len(cmd.Positional.Subs) > 0 {
-			return fmt.Errorf("help accepts a command, or '--all', but not both.")
+			return fmt.Errorf("help accepts a command, or '--all', but not both")
```

Comment:
So, in the other repo, this was actually deemed *not* to be a problem, so I reverted the change there. WDYT?

Reply:
It's not a problem, but "error messages with the '.'" is our usual style too, and none of the other `fmt.Errorf` messages in Pebble end in a `.`, so I "fixed" it.